### PR TITLE
Move bill pay to top-level tab

### PR DESCRIPTION
**Pull request by msalcala11**

### CODE REVIEW

**Review by gabrielbazan7 — APPROVED** [review]
ACK

File: src/navigation/tabs/shop/ShopHome.tsx [hunk 1]
```diff
@@ -339,20 +339,25 @@ const ShopHome: React.FC<
...
-                      : ShopTabs.SHOP_ONLINE,
-                  );
                   if (tab.target.includes(ShopTabs.BILLS)) {
+                    tab.preventDefault();
                     dispatch(
                       Analytics.track('Bill Pay - Clicked Bill Pay', {
                         context: 'Shop Tab',
                       }),
                     );
+                    navigation.navigate('Tabs', {
```
Comment (cmgustavo) [inline src/navigation/tabs/shop/ShopHome.tsx:349]: I think that we can remove `Pay Bills`, in order to keep it simple.

Comment (msalcala11) [reply src/navigation/tabs/shop/ShopHome.tsx:349]: Thanks for the review Gustavo! `Pay Bills` will be removed from the Shop tab in a subsequent app release after existing Bill Pay users have been given a chance to get used to the new location of the Bill Pay feature.

**Review by cmgustavo — DISMISSED** [review]
ACK!!

**Pull request merged**